scripts/gatsby-source-iiif/indexSearch.js

```diff
@@ -102,18 +102,18 @@ const getSearchDataFromManifest = (manifest) => {
     search['formatTag'] = [manifest.workType]
   }
 
-  search['allMetadata'] = ''
-  search['allMetadata'] = creators.join(' ')
+  search.allMetadata = []
+  search.allMetadata.push(creators.join(' '))
   allMetadataKeys.forEach((key) => {
     if (manifest[key]) {
-      search['allMetadata'] += ' ' + manifest[key]
+      search.allMetadata.push(manifest[key])
     }
   })
-  search['allMetadata'] += ' ' + loadSubItemTitles(manifest)
-  search['allMetadata'] += ' ' + search.repository
-  search['allMetadata'] += ' ' + search.centuryTag.join(' ')
-  search['allMetadata'] += ' ' + search.themeTag.join(' ')
-
+  search.allMetadata.push(loadSubItemTitles(manifest))
```

Comment:
how does changing the data type to an array change how the highlights come back?